clickhouse-cli-client/src/test/java/com/clickhouse/client/cli/ClickHouseCommandLineClientTest.java

```diff
@@ -190,4 +189,14 @@ public void testTransactionTimeout() throws ClickHouseException {
     public void testRowBinaryWithDefaults() throws ClickHouseException, IOException, ExecutionException, InterruptedException {
         throw new SkipException("Skip due to supported");
     }
+
+    @Test(dataProvider = "loadBalancingPolicies", groups = {"unit"})
```

Comment:
I'm not sure about failover for CLI client. Currently it does not work. So the tests are ignored. Please, let me know if failover should work for CLI client as well